### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by fniephaus**
and copy documentation from https://github.com/oracle/graal/blob/f237c4296bae102cd6983ab4f4e0b2f465712a4f/compiler/README.md#building-jvmci-jdk-8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# graal-jvmci-8
+
+Fork of jdk8u/hotspot with support for JVMCI.
+
+
+## Building JVMCI JDK 8
+
+To create a JVMCI enabled JDK 8 on other platforms (e.g., Windows):
```
Comment (dougxc) [inline README.md]: other?

Comment (fniephaus) [reply README.md]: I literally copied the section over...will fix this.

Comment (dougxc) [reply README.md]: Ok, great. This document needs to read correctly in a stand alone context.

Comment (fniephaus) [reply README.md]: PTAL

Comment (dougxc) [reply README.md]: ?

Comment (fniephaus) [reply README.md]: Oh sorry, [PTAL](https://en.wiktionary.org/wiki/PTAL).

Comment (dougxc) [reply README.md]: Aren't abbreviations a wonderful time saver ;-)

**Comment by dougxc** [comment]
looks good to me

**Pull request merged**